src/api/app/views/webui/package/_buildstatus.html.erb

```diff
@@ -5,17 +5,21 @@
       <% previous_repo = nil %>
       <% results.each do |result| %>
         <% repository = @project.repositories.find_by_name(result.repository) %>
-        <% next unless repository %>
-        <% next unless repository.architectures.pluck(:name).include?(result.architecture) %>
-        <tr>
+        <% is_outdated = !(repository && repository.architectures.pluck(:name).include?(result.architecture)) %>
+        <tr class="<%= is_outdated ? 'outdated_buildresult' : '' %>">
+          <td title="<%= result.repository %>" class="no_border_bottom <%= (result.repository == previous_repo) ? " no_border_top" : " "%>">
```

Comment:
I don't like removing the `rownspan` and hiding the rows border instead. And also it look strange that the whole row is gray, specially if it is the first architecture the one that is outdated. What about changing the style only of the `arch` and the `status`? Then you don't have to change the table structure neither.